LSPS2/README.md

```diff
@@ -582,13 +582,13 @@ If the `payment_size_msat` is specified in the request, the LSP:
 
 The following errors are specified for `lsps2.buy`:
 
-* `invalid_opening_fee_params` (2) - the `valid_until` field
+* `invalid_opening_fee_params` (200) - the `valid_until` field
```

Comment:
Nit: We use the error-code 200-twice.

I think we have enough space so we don't have re-use numbers. This makes it a bit simpler to implement a response-parser.

Reply:
Done